srml/balances/src/lib.rs

```diff
@@ -345,6 +345,21 @@ decl_module! {
 		/// of the transfer, the account will be reaped.
 		///
 		/// The dispatch origin for this call must be `Signed` by the transactor.
+		///
+		/// # <weight>
+		/// - Dependent on arguments but not critical, given proper implementations for
+		///   input config types. See related functions below.
+		/// - It contains a limited number of reads and writes internally and no complex computation.
+		///
+		/// Related functions:
```

Comment:
I agree that maintenance is annoying for such things but since it is (I think) the only occurrence of downstream references and not likely to change, I'd leave it be.